addons/remove-curved-stage-border/addon.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Remove Curved Stage Border",
+    "description": "Removes the curved border around the stage, allowing you to see the very corners.",
+    "tags": [
+        "editor"
+    ],
+    "credits": [
+        {
+            "name": "Tacodude7729",
+            "link": "https://github.com/Tacodude7729"
+        }
+    ],
+    "enabledByDefault": false,
+    "userstyles": [
+        {
+            "url": "remove-borders.css",
+            "matches": [
+                "https://scratch.mit.edu/projects/*/",
+                "https://scratch.mit.edu/projects/*/editor",
+                "https://scratch.mit.edu/projects/*/fullscreen",
+                "https://scratch.mit.edu/projects/editor"
```

Comment:
just do `https://scratch.mit.edu/projects/*`, the rest will be automatic

Reply:
> just do `https://scratch.mit.edu/projects/*`, the rest will be automatic

This will also match `https://scratch.mit.edu/projects/*/remixes` and `https://scratch.mit.edu/*/studios`, but in this case that isn't a problem.